tools/ci_build/github/linux/run_build.sh

```diff
@@ -32,6 +32,12 @@ elif [ $BUILD_DEVICE = "tensorrt" ]; then
         --use_openmp \
         --cuda_home /usr/local/cuda \
         --cudnn_home /usr/local/cuda --build_shared_lib $BUILD_EXTR_PAR
+elif [ $BUILD_DEVICE = "ngraph" ]; then
+    python3 $SCRIPT_DIR/../../build.py --build_dir /build \
+        --config Debug Release --build_shared_lib \
```

Comment:
--build_shared_lib option is duplicated

Reply:
Actually, it is duplicated for all other configs, and so for me since I copy-pasted from other config with minor change. I can fix it for nGraph config.